slices/libtbb12.yaml

```diff
@@ -0,0 +1,19 @@
+package: libtbb12
+
+essential:
+  - libtbb12_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+      - libtbbbind-2-5_libs
+      - libtbbmalloc2_libs
+    contents:
+      /usr/lib/*-linux-*/libtbb.so.12*:
```

Comment:
I just ran `ldd` on the shared object and found some intriguing details on an `amd64` machine:

```sh
root@ab160fb76645:~# ldd libtbb12/usr/lib/x86_64-linux-gnu/libtbb.so.12
        linux-vdso.so.1 (0x00007ffd8d1ba000)
        libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f2a05fef000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f2a05fc2000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f2a05db0000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f2a062c4000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f2a05cc7000)
```

It seems like the `libtbbbind-2-5` and `libtbbmalloc2` libraries are not a dependency here. Are they deps on other architectures?

Note: the changelog symlinks to that of `libtbbmalloc2`.
```sh
lrwxrwxrwx root/root         0 2023-12-27 14:25 ./usr/share/doc/libtbb12/changelog.Debian.gz -> ../libtbbmalloc2/changelog.Debian.gz
```

Reply:
Great you spotted the `dlopen` dynamic loads @clay-lake, I'll leave the changes unchanged then, if you agree @rebornplusplus.

Reply:
@rebornplusplus, may I ask you to elaborate on what you wrote about `Note: the changelog symlinks to that of libtbbmalloc2` ? I'm not sure I understand. The copyrights are correctly sliced and I thought in chiseled mode we are not considering the changelogs...

Reply:
Yeah this looks good, thanks Clay.

> @rebornplusplus, may I ask you to elaborate on what you wrote about `Note: the changelog symlinks to that of libtbbmalloc2` ? I'm not sure I understand. The copyrights are correctly sliced and I thought in chiseled mode we are not considering the changelogs...

Ah, this is nothing. I just noted it there because I was wondering if this symlink was the reason to include `libtbbmalloc2` as a dependency.